framework/src/main/java/org/checkerframework/common/value/ValueTransfer.java

```diff
@@ -1536,24 +1536,29 @@ protected TransferResult<CFValue, CFStore> strengthenAnnotationOfEqualTo(
 
     @Override
     protected void processConditionalPostconditions(
-            MethodInvocationNode n,
+            Node n,
```

Comment:
This change shouldn't be necessary - therefore the whole file doesn't need to change.

Reply:
Thanks! Done.